pb/payload.proto

```diff
@@ -71,3 +72,12 @@ message GenerateID {
 	bytes public_key       = 1;
 	int64 registration_fee = 2;
 }
+
+message NanoPay {
+	bytes sender    = 1;
+    bytes recipient = 2;
```

Comment:
The indent seems to be mixed with tab and spaces

Reply:
done